src/style.css

```diff
@@ -0,0 +1,69 @@
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+}
+
+.header__logo {
+  margin: 10px 95px 10px 51px;
+}
```

Comment:
you should use flex properties to move your logo, not margins

Reply:
Unfortunatly, when i use 
display: flex;
justify-content: center;
align-items: center;
It doesn't work on tests